### ISSUE TITLE
Angular 16, Templates are ignored

### ISSUE THREAD

**Issue by GarethPrattCv** [NileshPatel17/ng-multiselect-dropdown#405]
First of all, thank you for updating so that it's working in Angular 16!

**Angular version**:
16

**ng-multiselect-dropdown version**:
1.0.0

**Description of issue**:
The options and selected option templates no longer seem to have an effect as of Angular 16.

**Steps to reproduce**:
Create an ng-multi-select that uses option templates

**Expected result**:
Option text is as per the template

**Actual result**:
Option text is as if the template doesn't exist

**Demo**: Please share sample code link using StackBlitz or codesandbox
https://stackblitz.com/edit/stackblitz-starters-kb7s52?file=src%2Fmain.ts

**Comment by vandergil** [comment]
Hi, I'm also facing this issue!! 
Did you already solve it?

**Comment by garethrpratt** [comment]
> Hi, I'm also facing this issue!! Did you already solve it?

I'm afraid not @vandergil

**Comment by vandergil** [comment]
Tks!!
Do you have any other component similar to this one?

**Comment by garethrpratt** [comment]
No, we reverted to using this without templates. Let us know if you find a good alternative!

**Comment by vandergil** [comment]
Great! I can't do that because I am using a dropdown where I need to show a div showing the color and the color name.

I just found this one:
https://ng-select.github.io/ng-select#/multiselect 

But I haven't tested it yet and it's not as pretty as ng-multiselet-dropdown.
